<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSystemSettingDlg</class>
 <widget class="QDialog" name="CSystemSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
      <item>
       <widget class="QListWidget" name="sysSetting_listWidget">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="sysSetting_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="page_General">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>3</x>
               <y>40</y>
               <width>331</width>
               <height>25</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>日志文件路径</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_folderPath"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_select">
                <property name="text">
                 <string>选择</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="page_System">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>19</x>
               <y>23</y>
               <width>218</width>
               <height>22</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit_time">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>秒无人操作时，进入自动监控模式</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QCheckBox" name="exitWindowsCheck">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>80</y>
               <width>279</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>退出系统时提示用户</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_strategy">
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>实时数据策略</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>150</y>
               <width>81</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>历史数据策略</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>60</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>实时数据收集间隔</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_period">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>60</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_DataSource">
            <widget class="QLabel" name="label_18">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>100</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>实时数据服务</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_23">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>200</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>历史数据服务</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget_4">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>120</y>
               <width>251</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_25">
                <item>
                 <widget class="QLabel" name="label_29">
                  <property name="text">
                   <string>IP地址</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_RealIP">
                  <property name="text">
                   <string>192.168.200.183</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_26">
                <item>
                 <widget class="QLabel" name="label_30">
                  <property name="text">
                   <string>端口号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_RealPort">
                  <property name="text">
                   <string>1883</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="label_24">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>0</y>
               <width>111</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>路灯基础数据服务</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget_3">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>20</y>
               <width>251</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_21">
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>IP地址</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_LampIP">
                  <property name="text">
                   <string>192.168.200.129</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_22">
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>端口号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_LampPort">
                  <property name="text">
                   <string>5005</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>220</y>
               <width>251</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>IP地址</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_HistIP">
                  <property name="text">
                   <string>192.168.200.140</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>端口号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_HistPort">
                  <property name="text">
                   <string>5000</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="label_27">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>300</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>WSO2服务</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>330</y>
               <width>241</width>
               <height>170</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="5" column="2">
               <widget class="QLineEdit" name="lineEdit_httpsport">
                <property name="text">
                 <string>8243</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>HTTPS Port</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QLineEdit" name="lineEdit_MQTTPort">
                <property name="text">
                 <string>1886</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>MQTT端口号</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>HTTP Port</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLineEdit" name="lineEdit_httpport">
                <property name="text">
                 <string>8280</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLineEdit" name="lineEdit_password">
                <property name="text">
                 <string>admin</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="lineEdit_username">
                <property name="text">
                 <string>admin</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="lineEdit_WSO2Host">
                <property name="text">
                 <string>192.168.7.43</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>IP地址</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>444</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>应用</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
